src/routes/(console)/project-[project]/databases/database-[database]/+page.ts

```diff
@@ -3,22 +3,26 @@ import { sdk } from '$lib/stores/sdk';
 import { getLimit, getPage, getView, pageToOffset, View } from '$lib/helpers/load';
 import type { PageLoad } from './$types';
 import { CARD_LIMIT, Dependencies } from '$lib/constants';
+import { collections as collectionsStore } from './store';
 
 export const load: PageLoad = async ({ params, url, route, depends }) => {
     depends(Dependencies.COLLECTIONS);
     const page = getPage(url);
     const limit = getLimit(url, route, CARD_LIMIT);
     const view = getView(url, route, View.Grid);
     const offset = pageToOffset(page, limit);
+    const collections = await sdk.forProject.databases.listCollections(params.database, [
```

Comment:
can we check and see, in the layout of collections we also fetch documents - lets remove it there

Reply:
actually lets sync with design on this, we shouldnt be fetching collections without deleting 👍🏻